ssl/statem/extensions_clnt.c

```diff
@@ -15,11 +15,6 @@
 #ifndef OPENSSL_NO_ECH
 # include <openssl/rand.h>
 #include "internal/ech_helpers.h"
-/*
```

Comment:
When you are touching this file, could you also add one space between `#` and `include` on the above line?

Reply:
done